tests/test_cloudpy.py

```diff
@@ -108,6 +109,22 @@ def test_send_sms_success01(self):
         self.assertTrue('msgid' in xoxzo_res.message)
         self.assertEqual(xoxzo_res.messages,[])
 
+    def test_send_sms_kwargs_success01(self):
+        xoxzo_res = self.xc.send_sms(
+            "Hello from Xoxzo",
+            self.test_recipient,
+            self.test_sender,
+            foo="bar")
```

Comment:
there's no test for foo="bar". How to make sure, that kwargs works? Maybe you can pass any real kwarg? I'm not familiar, but I think you can pass a link tracking, or KDDI params. @anonaka .